scipy/spatial/transform/rotation.py

```diff
@@ -224,6 +223,7 @@ def __init__(self, quat, normalized=False):
             # Normalize each quaternion, ensuring norm is broadcasted along
             # each column.
             self._quat[~zero_norms] /= norms[~zero_norms][:, None]
+        self.num_rots = self._quat.shape[0]
```

Comment:
I suggest to make it property without setter.

Reply:
Can you explain what you mean by that? A property without setter?